pkg/restclient/common.go

```diff
@@ -0,0 +1,73 @@
+package restclient
```

Comment:
I prefer having all the external facing libraries under `glusterd2/pkg`.
With this patch I will keep it as `glusterd2/pkg/restclient`, we can change this in future if required.